### DISCUSSION TITLE
Better Categorization and Labeling of Operators

### DISCUSSION THREAD

**Discussion by jawight** [ReactiveX/rxjs#6953] [Ideas / Feature request]
It's not very clear with some operators whether they will modify emissions to other operators in the same pipe or just to observers. For example, Delay will delay emissions from the source observable until a given time or date. If there is a SwitchMap after delay in the pipe, delay will delay emissions of the new observable instead of the source. It would be good to note that in the delay definition and maybe even have a sub category for operators that behave this way like previously supported StartsWith.

**Comment by voliva** [comment]
As far as I know, delay doesn't "delay emissions of the new observable instead of the source" - That breaks composition. It just delays every emission of the source observable, so the following `switchMap` also receives the value delayed and so starts the new observable after it was received.

Do you have an example?
